app/views/home/_funding_levels.html.erb

```diff
@@ -0,0 +1,21 @@
+<section>
+  <h3>Fundraising Progress</h3>
+
+  <%= render "funding_progress_bar" %>
+
```

Comment:
I think maybe insert something like this... not sure if before or after the infographic is better.

```
In our first year, Ruby Together has completed significant improvements to the infrastructure that Ruby developers use every day:

- Shipped Bundler versions 1.10, 1.11, and 1.12, featuring conditional gems, optional groups, and a new gem index that can scale to handle the next five years of gems.
- Launched Gemstash 1.0, a server that can cache gems for your datacenter or office or host your private gems.
- Migrated RubyGems.org to use CDN servers around the world to serve data, speeding up gems for everyone.
```